api/condition_types.go

```diff
@@ -35,15 +35,16 @@ const (
 )
 
 const (
-	WaitingForDeploymentReason      = "WaitingForDeployment"
-	ExternalResourcesNotFoundReason = "ExternalResourcesNotFound"
-	DeploymentFailureReason         = "DeploymentFailure"
-	DeploymentUnavailableReason     = "DeploymentIsUnavailable"
-	RedeploymentExhaustedReason     = "AttemptToRedeployFailed"
-	WaitingForPlatformReason        = "WaitingForPlatform"
-	BuildFailedReason               = "BuildFailedReason"
-	WaitingForBuildReason           = "WaitingForBuild"
-	BuildIsRunningReason            = "BuildIsRunning"
+	WaitingForDeploymentReason           = "WaitingForDeployment"
+	ExternalResourcesNotFoundReason      = "ExternalResourcesNotFound"
+	DeploymentFailureReason              = "DeploymentFailure"
+	DeploymentUnavailableReason          = "DeploymentIsUnavailable"
+	RedeploymentExhaustedReason          = "AttemptToRedeployFailed"
+	WaitingForPlatformReason             = "WaitingForPlatform"
+	WaitingForConfigurationChangesReason = "WaitingForConfigurationChanges"
```

Comment:
Since the reason for the failed build is due to a wrong configuration we can myabe evalueate to rename it to `WrongConfigurationReason` that will more a little bit more meaningful.
It is just a consideration, it is not mandatory.

Reply:
ok